tests/test_maths/test_simpson_rule.py

```diff
@@ -0,0 +1,10 @@
+from maths import simpson_rule
+
+
+def test_simpson_rule():
+    a = 0.0
+    b = 1.0
+    steps = 10.0
+    boundary = [a, b]
+    y = simpson_rule.method_2(boundary, steps)
+    assert y == 0.33333333333333326
```

Comment:
I don't think you should compare floats like this. Use `pytest.approx` to assert that it is approximately 1 / 3.
```suggestion
    assert y == pytest.approx(1.0 / 3)
```